blocks/init/src/Blocks/components/image/manifest.json

```
@@ -4,23 +4,53 @@
 	"componentClass": "image",
 	"example": {
 		"attributes": {
-			"imageUrl": "https://raw.githubusercontent.com/infinum/eightshift-frontend-libs/develop/package/logo.svg?raw=true&sanitize=true",
-			"imageAlign": "center center",
-			"imageLink": "https://infinum.github.io/eightshift-docs/",
-			"imageAlt": "Some image description",
+			"imageUrl": {
+				"id": "1",
+				"url": "https://loremflickr.com/300/300"
```

```suggestion
				"url": "https://loremflickr.com/1000/800"
```

just because of the size of the screen 🙂